src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -45,6 +45,7 @@ public AdminController(WebScriptHostManager scriptHostManager, WebHostSettings w
         [HttpPost]
         [RequiresRunningHost]
         [Route("admin/functions/{name}")]
+        [DebugMode]
```

Comment:
nit: feels like the name would be clearer if it was something like `EnablesDebugMode` or something along those lines. Thoughts?

Reply:
Sure I can rename before checking in.

Reply:
Renamed